packages/yoroi-extension/app/containers/wallet/WalletSendPage.js

```diff
@@ -91,16 +104,11 @@ export default class WalletSendPage extends Component<InjectedOrGenerated<Genera
 
     const { transactionBuilderStore } = this.generated.stores;
 
-    const { uiDialogs, profile, } = this.generated.stores;
+    const { profile, } = this.generated.stores;
```

Comment:
Why this piece about blocking during a pending tx got removed? I don't see a mention of that in the PR description or in the task 🤔 

![image](https://user-images.githubusercontent.com/5585355/125702157-2fa49ba9-d5bd-450a-81d3-3c83dd6e5c7e.png)

Reply:
In discussion with @yushih now